package.json

```diff
@@ -33,10 +33,10 @@
     "release": "standard-version",
     "server": "serve docs --listen 3000",
     "start": "npm-run-all --parallel build:css watch:*",
-    "test": "npm run css-lint && npm run css",
+    "test": "npm-run-all css-lint css",
     "watch": "nodemon -e html,scss -x \"npm run css\"",
     "watch:eleventy": "eleventy --serve",
-    "watch:sass": "sass scss/:docs/assets/css/ --style=compressed --load-path=node_modules --source-map --watch"
+    "watch:sass": "npm run css-compile -- --watch"
```

Comment:
Again, except for `--watch` this is the same as `css-compile` - use it via composition instead.